index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,700;1,300;1,500&display=swap" rel="stylesheet">
+  <!-- Material Icons -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+  
+  <main class="main">
+ 
+<header class="nav-bar">    
+  <div class="logo-left">My Logo</div>
```

Comment:
- [ ] Kindly use the same color, font family, and font weight for you logo and menu button
### your design
![header](https://user-images.githubusercontent.com/81727988/170710680-146981be-54a9-42d7-a697-e06938f66cd7.png)

### Figma design
![figmaheader](https://user-images.githubusercontent.com/81727988/170710752-823dff9f-6cbf-4b88-ae22-a5b5445d1abf.png)